tests/test_models/test_engine/test_db_storage.py

```diff
@@ -86,3 +86,75 @@ def test_new(self):
     @unittest.skipIf(models.storage_t != 'db', "not testing db storage")
     def test_save(self):
         """Test that save properly saves objects to file.json"""
+
+    @unittest.skipIf(models.storage_t != 'db', "not testing db storage")
+    def test_get_none(self):
+        """Test that get retunrs None on not id or object not found"""
+        storage = DBStorage()
+        storage.reload()
+        for key in storage.all():
+            obj_type, dot, obj_id = key.partition('.')
+            break
+
+        obj = storage.get('', '')
+        self.assertEqual(obj, None)
```

Comment:
`TestCase.assertIsNone` already exists, so these checks can be simplified a little bit.